examples/nlp/text_classification/text_classification_with_bert.py

```diff
@@ -26,59 +26,50 @@
+parser.add_argument(
+    '--pretrained_model_name',
+    default='roberta-base',
+    type=str,
+    help='Name of the pre-trained model',
+    choices=nemo_nlp.nm.trainables.get_bert_models_list(),
+)
+parser.add_argument("--bert_checkpoint", default=None, type=str)
+parser.add_argument("--bert_config", default=None, type=str)
 parser.add_argument("--batch_size", default=32, type=int)
 parser.add_argument("--max_seq_length", default=36, type=int)
 parser.add_argument("--num_gpus", default=1, type=int)
 parser.add_argument("--num_epochs", default=10, type=int)
 parser.add_argument("--num_train_samples", default=-1, type=int)
 parser.add_argument("--num_eval_samples", default=-1, type=int)
+parser.add_argument("--optimizer_kind", default="adam", type=str)
 parser.add_argument("--lr_warmup_proportion", default=0.1, type=float)
 parser.add_argument("--lr", default=2e-5, type=float)
 parser.add_argument("--lr_policy", default="WarmupAnnealing", type=str)
+parser.add_argument("--amp_opt_level", default="O0", type=str, choices=["O0", "O1", "O2"])
 parser.add_argument("--weight_decay", default=0.01, type=float)
 parser.add_argument("--fc_dropout", default=0.1, type=float)
 parser.add_argument(
-    '--pretrained_model_name',
-    default='roberta-base',
-    type=str,
-    help='Name of the pre-trained model',
-    choices=nemo_nlp.nm.trainables.get_bert_models_list(),
-)
-parser.add_argument("--bert_checkpoint", default=None, type=str)
-parser.add_argument("--bert_config", default=None, type=str)
-parser.add_argument("--data_dir", required=True, type=str)
-parser.add_argument(
-    "--dataset_name",
-    required=True,
-    type=str,
-    choices=["default_format", "sst-2", "imdb", "thucnews", "jarvis", "nlu-ubuntu", "nlu-web", "nlu-chat"],
-    help="default_format assumes that a data file has a header and each line of the file follows "
-    + "the format: text [TAB] label. Label is assumed to be an integer.",
-)
-parser.add_argument(
-    "--use_cache", action='store_true', help="When specified loads and stores cache preprocessed data.",
+    "--use_cache", action='store_true', help="When specified loads and stores cache preprocessed data."
 )
 parser.add_argument("--train_file_prefix", default='train', type=str)
 parser.add_argument("--eval_file_prefix", default='test', type=str)
-parser.add_argument("--work_dir", default='outputs', type=str)
 parser.add_argument("--save_epoch_freq", default=1, type=int)
 parser.add_argument("--save_step_freq", default=-1, type=int)
 parser.add_argument('--loss_step_freq', default=25, type=int, help='Frequency of printing loss')
-parser.add_argument("--optimizer_kind", default="adam", type=str)
-parser.add_argument("--amp_opt_level", default="O0", type=str, choices=["O0", "O1", "O2"])
 parser.add_argument("--do_lower_case", action='store_true')
-parser.add_argument("--no_shuffle_data", action='store_false', dest="shuffle_data")
+parser.add_argument("--shuffle_data", type=bool, default=True)
```

Comment:
please revert back to `--no_shuffle_data", action='store_false', dest="shuffle_data`

Reply:
change it back.